onnxruntime/core/framework/debug_node_inputs_outputs_utils.h

```diff
@@ -51,7 +59,8 @@ struct NodeDumpOptions {
     Shape = 1 << 0,
     InputData = 1 << 1,
     OutputData = 1 << 2,
-    AllData = Shape | InputData | OutputData,
+    NodeMeta = 1 << 3,
```

Comment:
could we call it NodeMetadata?

Reply:
I went with NodePlacement since that's the only data not redundant with ONNX graph. If more info would be useful in the future, we could rename it